### PR TITLE
Parallel Move Shard

### PR DESCRIPTION
**Pull request by maierlars**

### CODE REVIEW

File: 3.7/release-notes-new-features37.md [hunk 1]
```diff
@@ -8,3 +8,10 @@ Features and Improvements in ArangoDB 3.7
 The following list shows in detail which features have been added or improved in
 ArangoDB 3.7. ArangoDB 3.7 also contains several bug fixes that are not listed
 here.
+
+
+Cluster
+-------
+
+### Parallel Move Shard
+Shards can now move in parallel. The old locking mechanism was replaced by a read-write-lock and thus allows multiple jobs for the same _to_ server. The actual transfere rates are still limited on dbserver side but there is a huge overall speedup. This also affects `CleanOutServer` and `ResignLeadership` jobs.
```
Comment (jsteemann) [inline 3.7/release-notes-new-features37.md]:
```suggestion
Shards can now move in parallel. The old locking mechanism was replaced by a read-write-lock and thus allows multiple jobs for the same _to_ server. The actual transfer rates are still limited on dbserver side but there is a huge overall speedup. This also affects `CleanOutServer` and `ResignLeadership` jobs. 
```

**Comment by Simran-B** [comment]
@maierlars I changed `to server` into `destination server`, or is that incorrect?

**Comment by maierlars** [comment]
> @maierlars I changed `to server` into `destination server`, or is that incorrect?
I approve that change.

**Comment by maierlars** [comment]
image

**Pull request merged**